cmd/validate/image.go

```diff
@@ -374,6 +376,9 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 	cmd.Flags().StringVarP(&data.input, "json-input", "j", data.input,
 		"JSON representation of an ApplicationSnapshot Spec")
 
+	cmd.Flags().StringVarP(&data.snapshotSource, "snapshot-source", "a", data.snapshotSource,
```

Comment:
What if this was called `images`? I'm thinking how `ApplicationSnapshot` means absolutely nothing to anyone outside RHTAP. We may want to not mention that here at all. But that's a different PR. I'm just trying to make the case for not adding a new parameter named after `ApplicationSnapshot`. (Descriptions can be changed later on with ease, parameter names not so much.)

NOTE: We don't actually have to have a long and short version of the parameter name. Allowing just the long version is perfectly ok.